src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,75 @@
  */
 
 public class Plane {
+    private String model;
+    private String color;
+    private int length;
+    private int wingsWidth;
+    private int enginesCount;
 
+    private Plane(PlaneBuilder builder) {
+        this.model = builder.newPlane.model;
+        this.color = builder.newPlane.color;
+        this.length = builder.newPlane.length;
+        this.wingsWidth = builder.newPlane.wingsWidth;
+        this.enginesCount = builder.newPlane.enginesCount;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getLength() {
+        return length;
+    }
+
+    public int getWingsWidth() {
+        return wingsWidth;
+    }
+
+    public int getEnginesCount() {
+        return enginesCount;
+    }
+
+    public static class PlaneBuilder {
+        private Plane newPlane;
+        private PlaneBuilder builder;
```

Comment:
what fields should builder contain?

Reply:
wrong. only plane's field copies, no Plane and PlaneBuilder at the builder's fields